charts/device-plugin-operator/README.md

```diff
@@ -0,0 +1,27 @@
+# Intel Device Plugins Operator Helm Chart
+
+## Get Helm Repository Info
+```
+helm repo add intel https://intel.github.io/helm-charts/
+helm repo update
+```
+
+## Install Helm Chart
+In order to install device plugin Operator, its CRDs should be installed. 
+
+```
+helm install device-plugin-operator intel/device-plugin-operator [flags] 
+```
+You can then run helm search repo intel to see pulled charts.
+
+## Upgrade Chart
+```
+helm upgrade device-plugin-operator intel/device-plugin-operator [flags]
+```
+
+## Uninstall Chart
+```
+helm uninstall device-plugin-operator
+```
+
+
```

Comment:
Add a section which document the variables, like in the other charts.

Reply:
What parameters should I define for the operator?